tests/io/test_pyarrow_visitor.py

```diff
@@ -209,9 +209,9 @@ def test_pyarrow_variable_binary_to_iceberg() -> None:
 def test_pyarrow_struct_to_iceberg() -> None:
     pyarrow_struct = pa.struct(
         [
-            pa.field("foo", pa.string(), nullable=True, metadata={"field_id": "1", "doc": "foo doc"}),
-            pa.field("bar", pa.int32(), nullable=False, metadata={"field_id": "2"}),
-            pa.field("baz", pa.bool_(), nullable=True, metadata={"field_id": "3"}),
+            pa.field("foo", pa.string(), nullable=True, metadata={PYARROW_FIELD_ID_KEY: "1", "doc": "foo doc"}),
```

Comment:
Since it's in the test, shall we explicitly write `PARQUET:field_id` here instead of using the pre-defined string?

Reply:
Sounds good, updated 👍